packages/react/src/lavadome.jsx

```diff
@@ -48,15 +56,16 @@ function tokenToText(token) {
     return text;
 }
 
-export const LavaDome = ({ text, unsafeOpenModeShadow }) => {
-    // variable @text is named that way only for visibility - in reality it's a lavadome token
-    const token = text, host = useRef(null);
+export const LavaDome = ({ token, unsafeOpenModeShadow }) => {
+    const host = useRef(null);
```

Comment:
what does this do?

Reply:
to be clear, I'm asking about L60 not L59

Reply:
`useRef` is this React feature that allows you to create a reference to some value that should not trigger rerendering of a React component (since other types of variables definition in React do trigger a rerender).

It's useful here, because React allows you to make it point to a non React declarative DOM node, so that it's accessible when passed on to another component (which is what we do with the `span` host here).